app/locations/api.py

```diff
@@ -1,18 +1,57 @@
-from .service import list_POI, get_details_for_POI, new_POI_suggestion
 from flask import jsonify
-from .errors import POINotFoundError, InvalidPOISuggestionError
-from typing import Dict
+from typing import Dict, Any
+
 from app.auth import with_auth_user
 from app.user.user import User
+from app.base_api_error import MissingQueryParameterError
+from .poi import POIClassification, POI
+from .service import list_POI, get_details_for_POI, new_POI_suggestion
+
+
+def _build_POI_api_model(
+    poi: POI, estimate: float, distance: float, last_updated: float
+) -> Dict[str, Any]:
+    """
+    Build a model to be returned by the POI api. Extends the POI dictionary with additional fields.
+    """
+    return {
+        "_id": poi.id,
+        "name": poi.name,
+        "type": poi.poi_type,
+        "class": poi.classification.value,
+        "estimate": estimate,
+        "distance": distance,
+        "lastUpdated": last_updated,
+    }
 
 
 @with_auth_user
 def get_all_POI(**kwargs):

```

Comment:
It may be better to explicitly accept latitude and longitude in the function declaration so it's obvious to whoever is reading the code what required parameters the function needs